getTokenIds-test.js

```diff
@@ -1,26 +1,23 @@
 const { connect, getRedisClient, getRedisItem, putRedisItem } = require("./redis");
 const { redisPrefixes } = require("./constants.js");
-const { getTokenIDs } = require("./tokens");
-const config = require("./config")["networks"]["rinkeby"];
+const { getTokenIDs, getTokenURIs } = require("./tokens");
+const config = require("./config").networks.rinkeby;
 
 async function main() {
     await connect();
-    let tokenIdOwners = await getTokenIDs(config.contractName)(config.network);
-    let tokenOwnersIds = {};
-    let tokenIds = Object.keys(tokenIdOwners);
-    let owners = [...new Set(Object.values(tokenIdOwners))];
-    owners.forEach(async (owner) => {
-        tokenIds.forEach((tokenId) => {
-            if (tokenIdOwners[tokenId] == owner) {
-                if (!tokenOwnersIds[owner]) {
-                    tokenOwnersIds[owner] = [];
-                }
-                tokenOwnersIds[owner].push(tokenId);
-            }
-        });
-        await putRedisItem(owner, tokenOwnersIds[owner], redisPrefixes[config.contractName]);
-        let res = await getRedisItem(owner, redisPrefixes[config.contractName]);
-        console.log(res);
+    let tokenIdOwners = await getTokenIDs(config.contractName)("rinkeby");
+    const tokenIDs = Object.keys(tokenIdOwners);
+    const uris = await getTokenURIs("rinkeby", tokenIDs);
+    uris.forEach(async uri => {
+        try {
+            await putRedisItem(`${uri.tokenId}`, `${uri.tokenURI}`, redisPrefixes[config.contractName]+'uris');
+        } catch (error) {
+            console.log(error);
+        }
     });
+    console.log(redisPrefixes[config.contractName]+'uris');
+    fetchedURI = Object.values((await getRedisItem('2', redisPrefixes[config.contractName]+'uris')).Item).join('');
```

Comment:
This is a big mess. I have no idea what this is supposed to be, or what these mysterious constants and coersions are.

Can this be separate lines?